tests/components/google_mail/test_services.py

```diff
@@ -57,13 +58,23 @@ async def test_set_vacation(
     assert len(mock_client.mock_calls) == 5
 
 
+@pytest.mark.parametrize(
+    ("side_effect", "expected_exception"),
+    (
+        (RefreshError, RefreshError),
```

Comment:
The service call should not leak `RefreshError` or some other external exception, but raise `HomeAssistantError`. Probably catch exceptions in the service handlers and raise HomeAssistantError from there eg via a decorator.